versions.cfg

```diff
@@ -339,10 +338,13 @@ zest.pocompile = 1.5.0
 
 [versions:python3]
 jeepney = 0.4.3
+feedparser = 6.0.8
```

Comment:
@ale-rt I added the feedparser also to ``[versions:python3]`` because it has a dependency on sgmllib3k, which feedparser version 5.2.1 doesn't have.
I felt it's better to put feedparser also in that versions/py3 section instead of having feedparser in the normal section and it's direct dependency which doesn't exist on python2 somewhere else.
But this feeling probably doesn't hold much arguments, so I'm good to move feedparser in the main ``versions`` section.

Reply:
I am fine keeping it there. But let's wait for jenkins

Reply:
I am not sure which of the two approaches is more consistent. Probably there are already several Py2-only and Py3-only pins in the main versions section.

I ran `tox -c release/tox.ini -e constraints2,constraints3,constraints`, which is what I use as part of creating a new release, and the result seems fine:

```
$ grep feedparser release/constraints*
release/constraints.txt:feedparser==5.2.1; python_version < "3.0"
release/constraints.txt:feedparser==6.0.8; python_version >= "3.0"
release/constraints2.txt:feedparser==5.2.1
release/constraints3.txt:feedparser==6.0.8
```

Question is whether we need the new pin in 5.2, because this Plone version does not support Python 3.9.
But the previous [feedparser 5.2.1](https://github.com/kurtmckee/feedparser/blob/5.2.1/setup.py) is from 2015 and officially supports only Python 2.7-3.4. So a new pin on Python 3 seems fine.

Thanks!